src/Node.js

```diff
@@ -85,7 +85,7 @@ export default class Node extends ManagedNode {
      * @returns {this}
      */
     setCert(ledgerId) {
```

Comment:
Perhaps change this to require `ledgerId` to be of type `NetworkName` instead?

Reply:
changed to use networkName only not ledgerId to convert to networkName